devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -89,14 +89,13 @@ def parse(String description) {
 }
 
 private Map parseIasButtonMessage(String description) {
-    int zoneInt = Integer.parseInt((description - "zone status 0x"), 16)
-    if (zoneInt & 0x02) {
-        resultMap = getButtonResult('press')
-    } else {
-        resultMap = getButtonResult('release')
+    def zs = zigbee.parseZoneStatus(description)
+
+    if (zs == null) {
```

Comment:
I don't think this null check is necessary because this function is only called if the description starts with 'zone status'